roles/istio/tasks/create-gateway-ssl-secret.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This would need to be ansibleized. Worth checking if there is an Ansible modules that will help to generate ssl certificates.

Reply:
https://docs.ansible.com/ansible/latest/collections/community/crypto/index.html